src/Symfony/Component/Security/Http/Attribute/CurrentUser.php

```diff
@@ -11,10 +11,17 @@
 
 namespace Symfony\Component\Security\Http\Attribute;
 
+use Symfony\Component\HttpKernel\Attribute\ValueResolver;
+use Symfony\Component\Security\Http\Controller\UserValueResolver;
+
 /**
  * Indicates that a controller argument should receive the current logged user.
  */
 #[\Attribute(\Attribute::TARGET_PARAMETER)]
-class CurrentUser
+class CurrentUser extends ValueResolver
 {
+    public function __construct(bool $pin = false, bool $disabled = false)
```

Comment:
the argument is named `disable` in the parent, not `disabled`. This should stay consistent.